### PR TITLE
docs(bitrue): createOrder - update docstring

### PR DESCRIPTION
**Pull request by samgermain**

### CODE REVIEW

File: ts/src/bitrue.ts [hunk 1]
```diff
@@ -1308,6 +1308,12 @@ export default class bitrue extends Exchange {
          * @param {float} amount how much of currency you want to trade in units of base currency
          * @param {float} [price] the price at which the order is to be fullfilled, in units of the quote currency, ignored in market orders
          * @param {object} [params] extra parameters specific to the bitrue api endpoint
+         * @param {float} [params.triggerPrice] the price at which a trigger order is triggered at
+         *
+         * EXCHANGE SPECIFIC PARAMETERS
+         * @param {string} newClientOrderId a unique id for the order, automatically generated if not sent
+         * @param {decimal} icebergQty
+         * @param {long} recvWindow
```
Comment (carlosmiei) [inline ts/src/bitrue.ts:1316]: @samgermain is recWindow specific to this endpoint or common to all private calls?

Also I think that for the sake of clarity, we should list these parameters as [params.icebertQty]` so the users know they go inside `params` and are optional

Comment (samgermain) [reply ts/src/bitrue.ts:1316]: > @samgermain is recWindow specific to this endpoint or common to all private calls?
> 
> Also I think that for the sake of clarity, we should list these parameters as `[params.icebertQty]` so the users know they go inside `params` and are optional

You're right, I missed that

Comment (samgermain) [reply ts/src/bitrue.ts]: > is recWindow specific to this endpoint or common to all private calls?

It looks like it's a `param` for all private calls, I don't see why that would be a reason to not include it here though. I've just been adding the exchange specific params in while also trying to unify the `triggerPrice`/`postOnly`/`timeInForce` params to make it easier to see if there's any other params that could be unified or anything else thats required ... I'll probably only spend time on it for the `createOrder` method

Comment (carlosmiei) [reply ts/src/bitrue.ts]: > I don't see why that would be a reason to not include it here though

@samgermain the same reason we don't include parameters like the `timestamp`, `signature` etc, they're not directly related to this method so it adds nothing having them

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**